.github/CODEOWNERS

```
@@ -0,0 +1,94 @@
+# This is a comment
+
+# Each line is a file pattern followed by one or more owners
+
+# These owners will be the default owners for everything in
+
+# the repo. Unless a later match takes precedence
+
+# @global-owner1 and @global-owner2 will be requested for
+
+# review when someone opens a pull request
+
+\* @ParkEunwoo
```

```suggestion
* @ParkEunwoo
```

이렇게만 쓰고 나머지는 전부 지워도됨

---

프리티어 세팅이 잘못된거 아니냐

---

아아아악